### PR TITLE
Create Beginner's Guide to the GUI tutorial

### PR DESCRIPTION
**Pull request by Timokleia**
Here is the branch with the tutorial I created. 
Please have a look and we can discuss how we proceed. 💪
- update, images moved to squarespace for saving space.

### CODE REVIEW

File: docs/labelling.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Selecting Frames to Label
+
+In DeepLabCut, choosing the right frames for labeling is a key step. You've got two easy ways to do this:
+
+1. **Let DeepLabCut Choose:** DeepLabCut can extract frames automatically for you. It's got two neat ways to do that:
+   - **Uniform:** This is like taking a snapshot at regular time intervals.
+   - **K-means:** This one looks for frames where things are happening differently, giving you a variety of actions and poses.
+
+2. **Pick Frames Yourself:** Just like flipping through a photo album, you can go through your video and pick the frames that catch your eye. Choose the **`automatic`** extraction method.
```
Comment (jeylau) [inline docs/labelling.md]:
```suggestion
2. **Pick Frames Yourself:** Just like flipping through a photo album, you can go through your video and pick the frames that catch your eye. Choose the **`manual`** extraction method.
```

Comment (jeylau) [reply docs/labelling.md]: I'd also add a sentence saying that it is important to ensure content diversity when manually picking frames to train robust models.

File: docs/Training-Evaluation.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jeylau) [inline docs/Training-Evaluation.md]:
```suggestion
```

File: docs/beginners-guide.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+> 
+> - **Mac:** 
+>   - Press `Cmd + Space` and search for 'Terminal'.
+
+### Activating DeepLabCut Environment
+
+If you have installed DeepLabCut via conda, activate the environment with the following command:
+
+```bash
+conda activate DeepLabCut
```
Comment (jeylau) [inline docs/beginners-guide.md]:
```suggestion
conda activate DEEPLABCUT
```

File: docs/beginners-guide.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+![DeepLabCut GUI Screenshot](https://github.com/Timokleia/DeepLabCut/blob/create-tutorial/docs/images/GUI-screenshot.png?raw=true)
+
+> 💡 **Note:** For a visual guide on navigating through the DeepLabCut GUI, check out our [YouTube tutorial](https://www.youtube.com/watch?v=tr3npnXWoD4).
+
+## Starting a New Project
+
+### Navigating the GUI on Initial Launch
+
+When you first launch the GUI, you'll come across three primary main options:
```
Comment (jeylau) [inline docs/beginners-guide.md]:
```suggestion
When you first launch the GUI, you'll find three primary main options:
```

File: docs/beginners-guide.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+   - A new folder named after your project's name will be created in the location you chose above.
+     
+
+### 📽 Video Tutorial: Setting Up Your Project in DeepLabCut
+
+![DeepLabCut Create Project GIF](https://github.com/Timokleia/DeepLabCut/blob/create-tutorial/docs/images/create-project.gif)
+
+
+
+
```
Comment (jeylau) [inline docs/beginners-guide.md]:
```suggestion
```

File: docs/labelling.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jeylau) [inline docs/labelling.md]:
```suggestion
```

**Review by jeylau — DISMISSED** [review]
Great, Tim! I've just suggested minor edits.

**Review by MMathisLab — APPROVED** [review]

**Review by MMathisLab — APPROVED** [review]

**Review by n-poulsen — APPROVED** [review]
LGTM!

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**